src/server.ts

```diff
@@ -15,29 +14,29 @@ const HOST = '0.0.0.0';
 const app = express();
 const parser = json();
 
-app.get("/ping", (req, res) => {
-  res.status(200).send();
+app.get("/", (req:any, res:any) => {
+  res.status(200).send("hello world!");
+});
+
+app.get("/ping", (req:any, res:any) => {
+  res.status(200).send("pong");
 });
 
-app.put('/local-item', parser, putItem);
 app.get('/local-item', parser, getItem);
-app.post('/local-item', parser, updateItem);
+app.post('/local-item', parser, putItem);
+app.put('/local-item/:id', parser, updateItem);
 app.delete('/local-item', parser, deleteItem);
 
 app.put('/dynamodb-item', parser, putDynamoDbItem);
 app.post('/dynamodb-item', parser, updateDynamoDbItem);
 app.get('/dynamodb-item', parser, getDynamoDbItem);
 app.delete('/dynamodb-item', parser, deleteDynamoDbItem);
 
-app.get('/postgresql-item/:id', parser, getPostgresDbItem);
-app.get('/postgresql-item', parser, getPostgresDbItem);
-app.put('/postgresql-item', parser, createPostgresDbItem);
-app.post('/postgresql-item/:id', parser, updatePostgresItem);
-app.delete('/postgresql-item/:id', parser, deletePostgresItem);
+app.get('/postgres-item/:id', parser, getPostgresDbItem);
+app.get('/postgres-item', parser, getPostgresDbItem);
```

Comment:
let's use two different function for these - getPotsgresDbItem and listPostgresDbItem.

Reply:
this is as requirements. we use the same function with or without the id. as you can see in the logic of the controller, we have different behaviors.

On the route side, we have to check the 2 cases in this way, otherwise we get an error if we use just the first one but we don't pass the id. if we use the second one, we will not get any parameter ID